OpenRA.Mods.RA/Traits/MadTank.cs

```diff
@@ -147,7 +147,7 @@ void StartDetonationSequence()
 
 			self.World.AddFrameEndTask(w => EjectDriver());
 			if (info.ThumpSequence != null)
-				renderUnit.PlayCustomAnimRepeating(self, info.ThumpSequence);
+				renderUnit.PlayCustomAnimationRepeating(self, info.ThumpSequence);
```

Comment:
This can now cache the generic ISpriteBody instead of RenderUnit.  You'll also need an ISpriteBodyInfo for the Requires<>.